JavaAssignments/src/amarja/Leapyear.java

```diff
@@ -0,0 +1,25 @@
+package amarja;
+import java.util.Scanner;
+//Q3 Program to find whether year input is leap year or not
+public class Leapyear {
+	boolean findLeapyear(int year)//Year is leap year if its divisble by 4 or its divsible by 100 and 400 evenly
```

Comment:
We need to discuss leap year condition.
according to your logic 2200 is a leap year, actually it is not.